tensorflow/core/kernels/conv_grad_input_ops.cc

```diff
@@ -497,10 +497,13 @@ void LaunchConv2DBackpropInputOp<GPUDevice, Eigen::bfloat16>::operator()(
     return;
   }
 
+  // Need to use the address of in_backprop rather than casted_in_backprop,
```

Comment:
On lines 468-470: Move the `casted_out_backprop` and `casted_filter` and `casted_in_backprop` to the `if (cast_to_float)` block to limit their scope. And similarly for other files.

Once done, I don't think this comment is necessary since `casted_in_backprop` won't be in scope.

Reply:
Done. PTAL. @reedwm